spec/test_app_templates/Gemfile.extra

```diff
@@ -1,4 +1,7 @@
 # placeholder to use for pinning against specific gem commit references
+gem 'hydra-pcdm', github:'projecthydra/hydra-pcdm', branch: 'master'
+gem 'hydra-works', github:'projecthydra/hydra-works', branch: 'master'
+gem 'active-fedora', '>= 10.3.0.rc1'
```

Comment:
Why do we need this line in `Gemfile.extra`?  This is already expressed in `curation_concerns.gemspec`